client/src/components/DisplayAllCampaigns.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Flex } from '@radix-ui/themes';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { settlementApi } from '../service/api';
+import { TypeServerSettlement } from '../../../SettlementTypes';
+
+const DisplayAllCampaigns = () => {
+    const navigate = useNavigate();
+    const [allCampaigns, setAllCampaigns] = useState<TypeServerSettlement[]>([]);
+
+    useEffect(() => {
+        axios
+            .get(`${settlementApi}/get`)
+            .then((res) => {
+                console.log(res.data);
+                setAllCampaigns(res.data.settlements);
+            })
+            .catch((err) => console.log(err));
+    }, []);
+
+    const handleNavigation = () => {
+        navigate('/twilight-library/dashboard/create-campaign');
```

Comment:
just inline this function, there's no reason to declare above

Reply:
deleted, artifact from cpy+paste file creation